components/provisional-en.html

```diff
@@ -93,9 +93,16 @@ <h3>Provisional feature removal first notice</h3>
 	</tr>
 	<tr>
 		<th>Template <code>.gc-pg-hlpfl</code></th>
-		<td>"Page success widget" design pattern to let users share their experience on the page.</td>
+		<td>
+			<p>"Page success widget" design pattern to let users share their experience on the page.</p>
+			<p>Feature has been revised and renamed to "page feedback form" (<code>.gc-opuft</code>):</p>
```

Comment:
@EricDunsworth Just wondering what made you choose `.gc-opuft` has a class name?

Reply:
I renamed it to that after chatting with @GormFrank and @Garneauma about it. PP considers this feature to be officially called On-Page User Feedback Tool (OPUFT) on their end.

Btw the ``gc-pg-hlpfl`` class wasn't actually tied to any CSS in the past and was named after a super outdated name for the feature. I can't recall exactly what it was - but I think it was meant to be an abbreviation for "page helpful".

We also didn't want the stable version of this feature to introduce CSS styles to the old class name and potentially risk messing-up the design of pre-existing provisional implementations.

Reply:
FWIW this feature has been renamed once again to "page feedback tool". So I've renamed all the ``opuft`` classes to ``pft``.